src/measure_theory/measurable_space.lean

```diff
@@ -783,7 +783,7 @@ lemma measurable_set.pi {s : set δ} {t : Π i : δ, set (π i)} (hs : countable
   measurable_set (s.pi t) :=
 by { rw [pi_def], exact measurable_set.bInter hs (λ i hi, measurable_pi_apply _ (ht i hi)) }
 
-lemma measurable_set.pi_univ [encodable δ] {t : Π i : δ, set (π i)}
+lemma measurable_set.univ_pi [encodable δ] {t : Π i : δ, set (π i)}
```

Comment:
I am not so sure about this change (here and elsewhere): in the lemma statement, `pi` comes before `univ`, so is there a rationale behind swapping the order in the lemma name?

Reply:
The idea is that usually we denote `set.pi` as `s.pi t`, so we have lemma names like `set.union_pi` to rewrite `(s ∪ s₁).pi t`. Unfortunately, the notation `univ.pi t` doesn't work (I'm not quite sure why Lean thinks this means `set.univ.pi`, while it goes well everywhere else), but that's why `pi univ t` is called `univ_pi` in `data/set/basic.lean`.